deployment/network-operator/README.md

```diff
@@ -101,9 +101,19 @@ $ kubectl -n network-operator get pods
 By default the network operator
 deploys [Node Feature Discovery (NFD)](https://github.com/kubernetes-sigs/node-feature-discovery)
 in order to perform node labeling in the cluster to allow proper scheduling of Network Operator resources. If the nodes
-where already labeled by other means, it is possible to disable the deployment of NFD by setting
-`nfd.enabled=false` chart parameter.
+where already labeled by other means (either deployed from master or deployed within another deployment), it is possible to disable the deployment of NFD by setting
+`nfd.enabled=false` chart parameter and make sure that the installed version is `v0.13.2` or newer and has NodeFeatureApi enabled.
 
+##### Deploy NFD from master with NodeFeatureApi enabled
```

Comment:
nit: Why "from master"? Maybe rename to "from upstream". WDYT?

Reply:
Done